tests/Application/config/sylius/grids/book.php

```diff
@@ -31,6 +32,11 @@
             null,
             ['author.nationality'],
         ))
+        ->addFilter(AttributeNationalityFilter::create(
+            AttributeNationalityFilter::class,
```

Comment:
it seems to be incorrect, or at least misleading

Reply:
Why ? We can use FQCN as name and also call the create method of the filter.

Reply:
Yes, this is bit strange... but I understand why.
The FQCN is the type/name of the filter.

Reply:
@vasilvestre Could you remember me why we don't use the Filter::create method instead?

Reply:
I did line 35